R/croquet-package.R

```diff
@@ -0,0 +1,9 @@
+#' @keywords internal
+"_PACKAGE"
+
+# allowing for the use of the dot when piping
+utils::globalVariables(".")
```

Comment:
is this for the base R pipe??

Reply:
this one is for the magrittr pipe, when we use the placeholder. Otherwise we get that global variables warning about the `.`